grr/server/grr_response_server/fleetspeak_utils.py

```diff
@@ -112,3 +113,34 @@ def GetLabelsFromFleetspeak(client_id):
       grr_labels.append(fs_label.label)
 
   return grr_labels
+
+
+def GetClientIdsFromFleetspeak() -> List[Text]:
```

Comment:
Please see my comment above. Listing all the clients is guaranteed to fail on large deployments, since the number of results will be too great.

Reply:
I'll reply to all concerns related to listing all client IDs in this thread https://github.com/google/grr/pull/832#discussion_r477534903.